stdlib/test/builtin/test_string.mojo

```diff
@@ -631,7 +631,55 @@ fn test_isspace() raises:
     assert_false(isspace(ord(".")))
 
 
+fn test_ascii_aliases() raises:
+    var whitespaces = String(" \n\t\r\f\v")
+    for i in range(len(whitespaces)):
+        assert_true(whitespaces[i] in String.WHITESPACE)
+
+    assert_true(String("a") in String.ASCII_LOWERCASE)
+    assert_true(String("b") in String.ASCII_LOWERCASE)
+    assert_true(String("y") in String.ASCII_LOWERCASE)
+    assert_true(String("z") in String.ASCII_LOWERCASE)
+
+    assert_true(String("A") in String.ASCII_UPPERCASE)
+    assert_true(String("B") in String.ASCII_UPPERCASE)
+    assert_true(String("Y") in String.ASCII_UPPERCASE)
+    assert_true(String("Z") in String.ASCII_UPPERCASE)
+
+    assert_true(String("a") in String.ASCII_LETTERS)
+    assert_true(String("b") in String.ASCII_LETTERS)
+    assert_true(String("y") in String.ASCII_LETTERS)
+    assert_true(String("z") in String.ASCII_LETTERS)
+    assert_true(String("A") in String.ASCII_LETTERS)
+    assert_true(String("B") in String.ASCII_LETTERS)
+    assert_true(String("Y") in String.ASCII_LETTERS)
+    assert_true(String("Z") in String.ASCII_LETTERS)
+
+    assert_true(String("0") in String.DIGITS)
+    assert_true(String("9") in String.DIGITS)
+
+    assert_true(String("0") in String.HEX_DIGITS)
+    assert_true(String("9") in String.HEX_DIGITS)
+    assert_true(String("A") in String.HEX_DIGITS)
+    assert_true(String("F") in String.HEX_DIGITS)
+
+    assert_true(String("7") in String.OCT_DIGITS)
+    assert_false(String("8") in String.OCT_DIGITS)
+
+    assert_true(String(",") in String.PUNCTUATION)
+    assert_true(String(".") in String.PUNCTUATION)
+    assert_true(String("\\") in String.PUNCTUATION)
+    assert_true(String("@") in String.PUNCTUATION)
+    assert_true(String('"') in String.PUNCTUATION)
+    assert_true(String("'") in String.PUNCTUATION)
+
+    var text = String("I love my Mom and Dad so much!!!\n")
+    for i in range(len(text)):
+        assert_true(text[i] in String.PRINTABLE)
```

Comment:
**Suggestion** This is good, but do you mind also adding a negative test (i.e. testing for a character that is not printable)?

Reply:
Done :)

Reply:
Thanks! I just re-imported this internally and it should land here in a little bit.